packages/swapper/src/swappers/utils/test-data/assets.ts

```diff
@@ -244,7 +244,7 @@ export const RUNE: Asset = {
   symbol: 'RUNE',
   precision: 8,
   color: '#33FF99',
-  icon: 'https://assets.coincap.io/assets/icons/rune@2x.png',
+  icon: 'https://coin-images.coingecko.com/coins/images/6595/large/Rune200x200.png?1696506946',
```

Comment:
Not super concerned about this because it's test data, but dis ERC20 RUNE

Reply:
Updated in [5fa1cb5](https://github.com/shapeshift/web/pull/7298/commits/5fa1cb537de3788088524efdd98220955cb62029)